receiver/pluginreceiver/rendered_config.go

```diff
@@ -48,6 +49,16 @@ type PipelineConfig struct {
 	Exporters  []string `yaml:"exporters,omitempty"`
 }
 
+// TelemetryConfig is a representation of collector telemetry settings
+type TelemetryConfig struct {
```

Comment:
@cpheps what is this config meant for exactly? Sounds like something to do with recording metrics?

Reply:
This is to mimic the [Telemetry Config](https://github.com/open-telemetry/opentelemetry-collector/blob/main/service/telemetry/config.go#L24). We can't use the configs in OTel as they don't have `yaml` struct tags. We need these tags as we "write out" the config before passing it into the new collector. This hardcoded telemetry config will be used to turn off telemetry in the new collector this receiver creates.

We actually did this in #379 but I think this change was lost in a later refactor.